room.js

```diff
@@ -1,703 +0,0 @@
-let room1_drawer_test_object = new box('drawer', false, false, 'none', 'drawer key', 'A wooden drawer slightly open with a note written on top!')
-let room1_drawer = {
-    opened: false,
-    current_room: 'room1',
-    current_facing: 'east',
-    visible: true,                                    
-    takeable: false,              // The key will only be takeable when it is visible and the chest in front of it has been opened
-    useable: true,
-    desc: "There is a wooden drawer slightly open with a note on top hastily written",
-    open(){
-        if(this.opened === false){
-            this.opened = true
-            this.desc = "The drawer is left opened like someone forgot to close it... looks sloppy."
-            console.log("\nYou found the drawer key!")
-            player.inv.push('drawer key')
-            console.log("Current inventory: ")
-            console.log(player.inv)
-        } else {
-            console.log("It's already open!")
-        }
-    },
-    close(){
-        if(this.opened === true){
-            this.opened = false
-            this.desc = "The drawer is now shut tight. Good for you keeping the place less tacky without a drawer hanging open awkwardly."
-        } else {
-            console.log("The drawer is already closed!")
-        }
-        
-    },
-    read(){
-        console.log('\n \x1b[36m%s\x1b[0m', `\n The note says \"The first digits are 457\"`)
-    },
-
-    take_note(){
-        console.log("You've taken the note!")
-    }
-}
-
-
-let room1_keypad = {
-    current_room: 'room1',
-    current_facing: 'north',
-    useable: true,
-    guesses: 3,
-    code: '45730',
-    desc: '\n A strange keypad with numbers 1-9. There\'s an LED text box above the numbers that says \"Waiting for input\"',
-    async use_keypad(){
-        keypad_answer = await ask("Enter code ")
-        if(keypad_answer === this.code){
-            console.log("Correct! The door unlocks")
-            player.isTrapped = false
-
-        } else if((keypad_answer != this.code) && this.guesses !== 0){
-            this.guesses -= 1
-            console.log(`\x1b[33m%s\x1b[0m: `, `\nWarning, incorrect answer! You have ${this.guesses} guesses left!`)
-        } else {
-            console.log(`\x1b[33m%s\x1b[0m: `, `\nRoom guesses have reached ${this.guesses}... Self-destruct initiated!`)
-            console.log("Game over!")
-            process.exit()
-        }
-    }
-}
-
-let room1_box_key = {
-    name: 'box key',
-    current_room: 'room1',
-    current_facing: 'east',
-    opened: false,
-    visible: true,                                    
-    takeable: false,              // The key will only be takeable when it is visible and the chest in front of it has been opened
-    useable: true, 
-    desc: '\nThis is the key you found in the box! It will come in use later...'
-}
```

Comment:
This is also another section of code that I don't believe is being used anywhere.